CondTools/RPC/interface/RPCCPPFLinkMapHandler.h

```diff
@@ -0,0 +1,40 @@
+#ifndef CondTools_RPC_RPCCPPFLinkMapHandler_h
+#define CondTools_RPC_RPCCPPFLinkMapHandler_h
+
+#include <string>
+#include <vector>
+
+#include "CondCore/PopCon/interface/PopConSourceHandler.h"
+#include "CondFormats/Common/interface/Time.h"
+
+#include "CondFormats/RPCObjects/interface/RPCAMCLinkMap.h"
+
+namespace edm {
+class ParameterSet;
+class ConfigurationDescriptions;
+} // namespace edm
+
+class RPCCPPFLinkMapHandler
+    : public popcon::PopConSourceHandler<RPCAMCLinkMap>
+{
+public:
+    RPCCPPFLinkMapHandler(edm::ParameterSet const & config);
+    ~RPCCPPFLinkMapHandler();
+
+    void getNewObjects();
+    std::string id() const;
+
+protected:
+    std::string id_;
+    std::string data_tag_;
+    cond::Time_t since_run_;
+
+    std::string input_file_;
+    std::vector<int> side_fed_;
+    unsigned int n_sectors_;
+    std::vector<std::vector<int> > side_sector_amc_;
```

Comment:
It could be, but I didn't use arrays because this handler is almost the same as `CondTools/RPC/interface/RPCTwinMuxLinkMapHandler.h`, where the size is hard-coded to 5.  The textual input format is used in online software too, and I'm hoping that changes there will make the CPPF/OMTF and TwinMux input txt files uniform enough to merge these into one handler.

These handler's will likely be run less than once a year, so performance-wise that won't influence things.

That said, if you prefer, I can change still change it.